app/services/users/geo_search.rb

```diff
@@ -112,18 +112,27 @@ def available_motifs_from_attributed_agents_arel
   end
 
   def available_motifs_base
-    @available_motifs_base ||= Motif.where(id: individual_motifs + collective_motifs).joins(:organisation)
+    @available_motifs_base ||= Motif.where(id: individual_motifs_ids + collective_motifs_ids).joins(:organisation)
   end
 
   def individual_motifs
     @individual_motifs ||= Motif.active.individuel.joins(:plage_ouvertures).distinct
   end
 
+  def individual_motifs_ids
+    @individual_motifs_ids ||= Motif.select(:id).active.individuel.joins(:plage_ouvertures).distinct.ids
```

Comment:
Pourquoi ne pas utiliser `individual_motifs` ?
```suggestion
    @individual_motifs_ids ||= individual_motifs.select(:id).ids
```

Reply:
Oh, bien vu!
C'est corrigé à présent